stock_landed_costs_analytic/tests/test_stock_landed_costs_analytic.py

```diff
@@ -0,0 +1,138 @@
+# Copyright 2020 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.tests.common import TransactionCase
+
+
+class TestStockLandedCostsAnalytic(TransactionCase):
+    def setUp(self):
+        super().setUp()
+        self.Product = self.env["product.product"]
+        self.Picking = self.env["stock.picking"]
+        self.LandedCost = self.env["stock.landed.cost"]
+        self.ProductCategory = self.env["product.category"]
+        self.Account = self.env["account.account"]
+
+        self.analytic_distribution_1 = dict(
+            {str(self.env.ref("analytic.analytic_agrolait").id): 100.0}
+        )
+        self.analytic_distribution_2 = dict(
+            {str(self.env.ref("analytic.analytic_asustek").id): 100.0}
+        )
+        self.picking_type_in = self.env.ref("stock.picking_type_out")
+        self.supplier_location = self.env.ref("stock.stock_location_suppliers")
+        self.customer_location = self.env.ref("stock.stock_location_customers")
+
+        self.account_1 = self.Account.create(
+            {
+                "name": "Account 1 test",
+                "code": "Account1",
+                "account_type": self.env["account.account"]
+                .search([], limit=1)
+                .account_type,
+                "reconcile": True,
+            }
+        )
+        self.account_2 = self.Account.create(
+            {
+                "name": "Account 2 test",
+                "code": "Account2",
+                "account_type": self.env["account.account"]
+                .search([], limit=1)
+                .account_type,
+                "reconcile": True,
+            }
+        )
+        self.category = self.ProductCategory.create(
+            {
+                "name": "Product Category Test",
+                "property_cost_method": "fifo",
+                "property_valuation": "real_time",
+            }
+        )
+        self.product = self.Product.create(
+            {
+                "name": "Product Test",
+                "type": "product",
```

Comment:
```suggestion
                "type": "consu",
                "is_storable": True,
```
Please mate, add this to fix tests.

Reply:
@luisDIXMIT Thanks Luis, Totally right. I already made it!